contrib/cirrus/build.sh

```diff
@@ -12,19 +12,16 @@ echo "local-test-integration Overlay"
 make STORAGE_DRIVER=overlay local-test-integration
 echo "local-test-integration Fuse-overlay"
 make STORAGE_DRIVER=overlay STORAGE_OPTION=overlay.mount_program=/usr/bin/fuse-overlayfs local-test-integration
+echo "local-test-unit"
```

Comment:
s/unit/integration/

or, maybe easier to `set -x` on the whole script?  Sometimes that can be too noisy though.
Another option could be to add a "print then do" helper in `lib.sh` similar to https://github.com/containers/storage/blob/master/hack/get_ci_vm.sh#L36 (but without `--background`).

Reply:
Added showrun.

Reply:
https://github.com/containers/storage/pull/354